packages/http-server/src/__tests__/server.oas.spec.ts

```diff
@@ -64,19 +61,10 @@ describe('server', () => {
     expect(payload).toHaveProperty('name');
   });
 
-  test('will return 500 with error when an undefined code is requested', async () => {
```

Comment:
Would be nice to have a _positive_ test in place for this one. E.g. 
`should return requested status code with an error message even if error code not defined`

Reply:
Oh ok, I see you've done it several lines below